src/utils/getDateAndTime.js

```diff
@@ -0,0 +1,3 @@
+export default function getDateAndTime(block_timestamp) {
+    return new Date(block_timestamp).toLocaleString('en-US', {dateStyle: 'short', timeStyle: 'short'})
```

Comment:
Rather than assume EN-US, we can use the browser's locale settings for this:
https://github.com/wojtekmaj/get-user-locale/blob/master/src/index.js